pkg/session_recording/recordings.jsx

```diff
@@ -249,7 +254,7 @@
                     <div className="row">
                         <div className="col-md-12">
                             <ol className="breadcrumb">
-                                <li><a>Session Recording</a></li>
+                                <li><a onClick={this.goBackToList}>Session Recording</a></li>
```

Comment:
I can reproduce it reliably. I'll try to describe steps here:

1. Open http://localhost:9090/session_recording
1. Click on a recording, wait for it to open
1. Resize the terminal horizontally
1. Click on "Session Recording" "breadcrumb" link, wait for the recording list to open
1. Click on the same recording, wait for it to open
1. Resize the terminal horizontally

Result: resizing doesn't stop even if the mouse button is released
Expected: resizing behaves as before, it stops when mouse button is released